### PR TITLE
Add services json-ld to vet center template

### PR DESCRIPTION
**Pull request by wullaski**
## Description

Adds services json-ld to vet center template.

https://app.mural.co/t/vagov6717/m/vagov6717/1657548339031/75fd9c37c635478edcbc7d5179d95b6d53d9c339?sender=dfafbb7c-1fe6-468b-880a-a3e3fa8ff9c2

## Testing done

https://validator.schema.org

## Screenshots
Screen Shot 2022-08-05 at 2 39 49 PM

Screen Shot 2022-08-05 at 2 39 30 PM

## Acceptance criteria
- [x] Services JSON-LD is present in page source
- [x] Services JSON-LD validates at validate.schema.org

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/site/layouts/vet_center.drupal.liquid [hunk 1]
```diff
@@ -172,6 +172,49 @@
                 %}
               {% endfor %}
             </div>
+            {% for healthService in fieldHealthServices %}
+              <script type="application/ld+json">
+                {
+                  "@context": "https://schema.org",
+                  "@type": "GovernmentService",
+                  "name": "{{ healthService.entity.fieldServiceNameAndDescripti.entity.name }}",
+                  "alternateName": "{{ healthService.entity.fieldServiceNameAndDescripti.entity.fieldVetCenterFriendlyName }}",
```
Comment (ryguyk) [inline src/site/layouts/vet_center.drupal.liquid:181]: I noticed this is sometimes empty. Is a blank here acceptable?

File: src/site/layouts/vet_center.drupal.liquid [hunk 2]
```diff
@@ -172,6 +172,49 @@
...
+                      "@type": "Language",
+                      "name": "English",
+                      "alternateName": "en"
+                    },
+                    "serviceLocation": {
+                      "@type": "GovernmentOffice",
+                      "name": "{{ title }}",
+                      "address": {
+                        "@type": "PostalAddress",
+                        "streetAddress": "{{ fieldAddress.addressLine1 }}",
```
Comment (ryguyk) [inline src/site/layouts/vet_center.drupal.liquid:208]: Does `addressLine2` need to be appended if populated (e.g. 123 1st Ave. **Suite 100**)? I'm not sure how specific this metadata needs to be for our concerns.

**Review by ryguyk — APPROVED** [review]
Approved with a couple questions. I don't know the answers, so perhaps this is all good to go. If not, do not merge until fixing.

File: src/site/layouts/vet_center.drupal.liquid (hunk 1 above)
Comment (wullaski) [reply src/site/layouts/vet_center.drupal.liquid:181]: I think so. [Both validators](https://developers.google.com/search/docs/advanced/structured-data) don't throw any errors or warnings on that field being empty.

File: src/site/layouts/vet_center.drupal.liquid (hunk 2 above)
Comment (wullaski) [reply src/site/layouts/vet_center.drupal.liquid:208]: This might be good to include. I'll add it.

**Pull request merged**